### ISSUE TITLE
fatal error: 'llvm/IR/Value.h' file not found

### ISSUE THREAD

**Issue by wongzigii** [lsegal/my_toy_compiler#27]
Hi @lsegal 
I met errors after commanded`g++ -o parser parser.cpp tokens.cpp main.cpp`.
Googled but no luck.

    In file included from parser.y:2:
    ./node.h:3:10: fatal error: 'llvm/IR/Value.h' file not found
    #include <llvm/IR/Value.h>
             ^
    1 error generated.
    In file included from tokens.l:3:
    ./node.h:3:10: fatal error: 'llvm/IR/Value.h' file not found
    #include <llvm/IR/Value.h>
             ^
    1 error generated.
    In file included from main.cpp:2:
    ./codegen.h:3:10: fatal error: 'llvm/IR/Module.h' file not found
    #include <llvm/IR/Module.h>
             ^
    1 error generated.
  

This is the output of `g++ -v` :

    Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
    Apple LLVM version 6.1.0 (clang-602.0.49) (based on LLVM 3.6.0svn)
    Target: x86_64-apple-darwin14.3.0
    Thread model: posix

**Comment by lsegal** [comment]
Should be resolved by #29.

**Comment by akash-vartak** [comment]
what does that mean?? ```#29``` ?
My error:
```
akash007@ubuntu:~/ToyLanguage$ g++ -o parser parser.cpp tokens.cpp main.cpp  
In file included from parser.y:2:0:
node.h:3:27: fatal error: llvm/IR/Value.h: No such file or directory
 #include <llvm/IR/Value.h>
                           ^
compilation terminated.
In file included from tokens.l:3:0:
node.h:3:27: fatal error: llvm/IR/Value.h: No such file or directory
 #include <llvm/IR/Value.h>
                           ^
compilation terminated.
In file included from main.cpp:2:0:
node.h:3:27: fatal error: llvm/IR/Value.h: No such file or directory
 #include <llvm/IR/Value.h>
                           ^
compilation terminated.
```

Help me please!!
